### DISCUSSION TITLE
best practice of the deployment of flinkcdc

### DISCUSSION THREAD

**Discussion by binaryoak** [apache/flink-cdc#2485] [Q&A]
hi folks,
Just want to understand that in production env, what would be the best practice to deploy flinkcdc?  the flink cluster should be deployed in application mode or session mode?  I see the examples from the official flinkcdc docs are all in session mode, yet from flink's doc, it recommends to use application mode in production env.

**Comment by andrewtran1314** [comment]
hi team, any updates on this?
